docs/Credits.md

```diff
@@ -1,21 +1,17 @@
-Some pre-trained models provided by other repository are used in some packages.
+Certain AutowareArchitectureProposal packages rely on pre-trained CNN models provided by other open source repositories.
+- tensorrt_yolo3
+    - The pre-trained models originate from [TRTForYolov3](https://github.com/lewes6369/TensorRT-Yolov3). 
+    - [Weights for the trained model](https://drive.google.com/drive/folders/18OxNcRrDrCUmoAMgngJlhEglQ1Hqk_NJ) (416 folder) are automatically downloaded during the build process.
 
- - tensorrt_yolo3 <br>
-The pre-trained models are provided in the following repository. The trained file is automatically downloaded when you build. <br>
-https://github.com/lewes6369/TensorRT-Yolov3 <br>
-\[Original URL] <br>
-Tranined file (416) : https://drive.google.com/drive/folders/18OxNcRrDrCUmoAMgngJlhEglQ1Hqk_NJ
 
-- traffic_light_fine_detector <br>
-A trained model in this package is based on the following .weights file and was fine-tuned with darknet by Tier IV. <br>
-\[Original URL] <br>
-https://pjreddie.com/media/files/yolov3.weights <br>
-After fine-tuning, the trained model is converted to ONNX file with the following script. <br>
-https://github.com/tier4/AutowareArchitectureProposal.iv/blob/master/src/perception/traffic_light_recognition/traffic_light_fine_detector/scripts/yolov3_to_onnx.py <br>
+- traffic_light_fine_detector
+    - The trained model in this package is based on the [pjreddie's YOLO .weights file](https://pjreddie.com/media/files/yolov3.weights) and was fine-tuned by Tier IV using [Darknet](https://github.com/pjreddie/darknet).
+    - After fine-tuning, the trained model is converted to ONNX file using a [Python script](https://github.com/tier4/AutowareArchitectureProposal.iv/blob/master/src/perception/traffic_light_recognition/traffic_light_fine_detector/scripts/yolov3_to_onnx.py).
```

Comment:
"to **an** ONNX file"

Reply:
Good catch. I have a feeling that this section is out of date (c.f the question that I directed to Saito-san in Japanese), and so I'm waiting for him to respond and then I'll update this properly.